app.py

```diff
@@ -4,9 +4,98 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+todos = [
+    {'id': 1, 'title': 'First Todo', 'description': 'do this tomorrow'},
+    {'id': 2, 'title': 'Second Todo', 'description': 'do this day after'},
+    {'id': 3, 'title': 'Third Todo', 'description': 'do this day after that'}]
+
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return 'Hello World! <a href=http://127.0.0.1:5000/todo> todos</a>'
+
+
+@app.route('/todo', methods=['GET'])
+@app.route('/todo/', methods=['GET'])
+def todolist():
+    return render_template('index.html', todos=todos)
+
+
+@app.route('/todo/{todoId}', methods=['GET'])
+@app.route('/todo/{todoId}/', methods=['GET'])
+def gettodoitem():
+    return render_template('index.html', todos=todos)
+
+
+@app.route('/todo/reset', methods=['GET'])
+def resettodos():
+    todos.append({})
+    todos.clear()
+    return 'success'
+
+
+@app.route('/todo', methods=['POST'])
+def addtotodolist():
+    todos.append({'id': 4, 'title': request.form['title'], 'description': request.form['description']})
+    return render_template('index.html', todos=todos)
```

Comment:
You should `return redirect('/todo')` instead.

- User will see the correct url in the browser
- Refreshing the page will not resubmit the form
- Avoids code duplication